pybossa/view/projects.py

```diff
@@ -3191,6 +3191,11 @@ def coowners(short_name):
 
             filters = {'enabled': True}
             users = user_repo.search_by_name(query, **filters)
+
+            if form.contact and form.contact.data.lower() == 'true':
+                # Filter contacts only to users that are enabled and assigned to this project or are sub-admin/admin.
+                users = [user for user in users if is_user_enabled_assigned_project(user, project) or user.id in project.owners_ids]
```

Comment:
IMO, only users that are part of `project.owners_ids` to be contacted and admins/subadmins not part of `project.owners_ids` need not be considered.

Reply:
Also, user in contact list when removed from `owners_ids` should also be removed from `contacts` list

Reply:
To keep this PR simple, perhaps we should allow the user to manage the lists independently. That is, after removing a co-owner, they would also need to remove the contact as well (two clicks), followed by saving.

Otherwise, if we choose to keep the contacts and co-owners synchronized, it may require modifying the Vue component to react to the 'X' button on both lists in the UI, in addition to including the filter in the server-side code. Perhaps, this could be postponed to a future ticket?

This would help keep the scope of the work involved in this PR contained. Thoughts?

Reply:
yes, having future ticket to handle follow up work to synchronize contacts with co-owners is fine. thanks!

Reply:
Fixed via https://github.com/bloomberg/pybossa/pull/776/files#diff-7fa4bf701959a15b1589099598a22b2352b452e47a90c7b981d5d340d8bc45dfR3240-R3243